app/templates/login.hbs

```diff
@@ -94,28 +94,30 @@
     onClose=closeModal
     translucentOverlay=true
   }}
-    <h2>Forgot password</h2>
+    <h2 class="modal-header">Forgot password</h2>
     {{#let forgotPassword as |task|}}
       <form onsubmit={{prevent-default (perform task)}}>
         <legend>
-          First we need the e-mail address on your account
+          Enter your email address below and we'll email the instruction for setting a new one.
         </legend>
         <div>
-          <label>email address:</label>
+          {{!-- <label>Email address:</label> --}}
```

Comment:
Is there a reason you're leaving this comment here?

Reply:
I didn't think of the github would show the file changes so was trying to comment the deleted content for your reference. But will not do that next time